src/styles/blocks/cards-display.scss

```diff
@@ -0,0 +1,26 @@
+@use '../utils/variables' as *;
+
+.cards-display {
+  padding: 50px 40px;
+
+  display: grid;
+  justify-content: center;
+
+  gap: 46px 48px;
+
+  @media (max-width: 487px) {
+    grid-template: auto / $card-width;
+  }
+
+  @media (min-width: 488px) {
+    grid-template: auto / repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template: auto / repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template: auto / repeat(4, $card-width);
```

Comment:
The `$card-width` variable is used here. Ensure that it is defined in the `variables` file to avoid any compilation errors.